src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/Cas1OutOfServiceBedEntity.kt

```diff
@@ -12,22 +12,49 @@ import javax.persistence.Entity
 import javax.persistence.Id
```

Comment:
just a tweak to the commit title is needed perhaps for ".. to use revision entity"?